Stanford/JumpstartAcademic/Install/Content/ImportJSAcademicVocabularies.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @file
+ * Abstract Task Class.
+ */
+
+namespace Stanford\JumpstartAcademic\Install\Content;
+/**
+ *
+ */
+class ImportJSAcademicVocabularies extends \AbstractInstallTask {
+
+  /**
+   * Set the site name.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+    $endpoint = "https://sites.stanford.edu/jsa-content/jsainstall";
```

Comment:
We should define this as a constant, if we're going to think about giving it a vhost or moving it to Anchorage. I'll make a ticket.